### DISCUSSION TITLE
/kuberentes-federation

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#476] [Announcements]
https://draveness.me/kuberentes-federation/

**Comment by zcong1993** [comment]
终于更新了！
第一句话 typo 从比较早的版本就**生成** -> 声称

**Comment by draveness** [comment]
> @zcong1993
> 终于更新了！
> 第一句话 typo 从比较早的版本就**生成** -> 声称

感谢支持啊，确实太久没更新了

**Comment by qiuyuyin** [comment]
左神勤政🥰

**Comment by zcong1993** [comment]
> @draveness
> > @zcong1993
> > 终于更新了！
> > 第一句话 typo 从比较早的版本就**生成** -> 声称
> 
> 感谢支持啊，确实太久没更新了

评论可以试试这个 https://giscus.app/zh-CN 对回复很友好

**Comment by wujunwei** [comment]
支持支持，这篇刚好讲到最近研究的内容

**Comment by imageslr** [comment]
@zcong1993 确实不错。不过现有的都是基于 issue 的评论，想要平移到 discussion 不太方便吧。giscus 这个插件不知道有没有迁移方案。

---

看了下是有的。

**Reply by draveness** [reply]
已经迁移了啊

**Comment by zaclau-dev** [comment]
nice ~

**Comment by edwardstudy** [comment]
为什么大佬总结kubefed的内容比我写的好呢？😂
还请教下，大佬有没有研究过liqo这套方案：通过virtual node的方式上报一个集群的资源。
https://edwardesire.com/posts/multi-cluster-scheduling/#liqo

**Comment by GeorgeGuo2018** [comment]
左神，谈到多集群，推荐下腾讯开源的clusternet。。。。

已经历过生产检验，架构上也比华为的karmada轻量很多

**Comment by jkrllezvous** [comment]
请问左神对 [Redis多线程怎么看](https://post.techwavelabs.org/redis/multi-thread-redis/) ?

**Comment by push-and-pop** [comment]
https://draveness.me/whys-the-design-mongodb-b-tree/
大佬    为什么mongo使用btree这篇文章看不了了

**Comment by welin-jens** [comment]
看了大佬的博客才知道为啥同一年毕业的人会有这么大的差距，受教了！

**Comment by hahazei** [comment]
作者好久没更新了

**Comment by PixelCores** [comment]
好久没更新了，不知道以后会不会继续更新？

**Comment by qzQi** [comment]
好久没更新了
